Description: Remove uses of deprecated GTK functions
 Port gtkpod away from deprecated functions in GTK to make it
 build with GTK 3.1 in Ubuntu.
Origin: vendor
Author: Maia Kozheva <sikon@ubuntu.com>
Forwarded: http://gtkpod.org/bugs/index.php?do=details&task_id=82
Last-Update: 2011-08-20
---
Index: gtkpod-2.1.0/plugins/sorttab_display/display_sorttabs.c
===================================================================
--- gtkpod-2.1.0.orig/plugins/sorttab_display/display_sorttabs.c	2011-08-20 22:04:36.000000000 +0700
+++ gtkpod-2.1.0/plugins/sorttab_display/display_sorttabs.c	2011-08-20 22:11:16.000000000 +0700
@@ -2300,7 +2300,7 @@
     for (i = 0; i < PANED_NUM_ST; ++i) {
         GtkWidget *paned;
 
-        paned = gtk_hpaned_new();
+        paned = gtk_paned_new(GTK_ORIENTATION_HORIZONTAL);
         gtk_widget_show(paned);
 
         if (!i) {
Index: gtkpod-2.1.0/plugins/sorttab_display/plugin.c
===================================================================
--- gtkpod-2.1.0.orig/plugins/sorttab_display/plugin.c	2011-08-20 22:04:36.000000000 +0700
+++ gtkpod-2.1.0/plugins/sorttab_display/plugin.c	2011-08-20 22:11:16.000000000 +0700
@@ -176,7 +176,7 @@
     set_default_preferences();
 
     /* Add widget in Shell. Any number of widgets can be added */
-    sorttab_display_plugin->st_paned = gtk_hpaned_new();
+    sorttab_display_plugin->st_paned = gtk_paned_new(GTK_ORIENTATION_HORIZONTAL);
     gchar *glade_path = g_build_filename(get_glade_dir(), "sorttab_display.xml", NULL);
     st_create_tabs(GTK_PANED(sorttab_display_plugin->st_paned), glade_path);
     gtk_widget_show(sorttab_display_plugin->st_paned);
Index: gtkpod-2.1.0/plugins/cover_display/display_coverart.c
===================================================================
--- gtkpod-2.1.0.orig/plugins/cover_display/display_coverart.c	2011-08-20 22:25:21.884692001 +0700
+++ gtkpod-2.1.0/plugins/cover_display/display_coverart.c	2011-08-20 22:26:26.804692979 +0700
@@ -174,7 +174,7 @@
     cdwidget->controlbox = gtkpod_builder_xml_get_widget(xml, "cover_display_controlbox");
     cdwidget->leftbutton = GTK_BUTTON (gtkpod_builder_xml_get_widget (xml, "cover_display_leftbutton"));
     cdwidget->rightbutton = GTK_BUTTON (gtkpod_builder_xml_get_widget (xml, "cover_display_rightbutton"));
-    cdwidget->cdslider = GTK_HSCALE (gtkpod_builder_xml_get_widget (xml, "cover_display_scaler"));
+    cdwidget->cdslider = GTK_SCALE (gtkpod_builder_xml_get_widget (xml, "cover_display_scaler"));
     /* create a new drawing area */
     cdwidget->draw_area = gtk_drawing_area_new();
     cdwidget->cdcovers = g_ptr_array_sized_new(IMG_TOTAL);
Index: gtkpod-2.1.0/plugins/cover_display/display_coverart.h
===================================================================
--- gtkpod-2.1.0.orig/plugins/cover_display/display_coverart.h	2011-08-20 22:23:56.884690722 +0700
+++ gtkpod-2.1.0/plugins/cover_display/display_coverart.h	2011-08-20 22:25:28.444692102 +0700
@@ -68,7 +68,7 @@
 	GtkWidget *canvasbox;
 	GtkWidget *controlbox;
 	GtkButton *leftbutton;
-	GtkHScale *cdslider;
+	GtkScale *cdslider;
 	GtkButton *rightbutton;
 
 	/* Utility data */
